### PR TITLE
Client add executors

### PR DESCRIPTION
**Pull request by agagancarczyk**
## Motivation
1. Saving changed client profiles.
2. Reloading client profiles.
3. Deleting newly created client profile.
4. Showing "Executors" text, "Add Executor" button and Action dropdown on new client form when a new client is created.
5. Adding cypress tests.

## Verification Steps
1. Go to Realm Settings > Client policies > Profiles > Json editor 
2. Change one of the existing profiles, click "Reload" btn and verify that original profiles display
2. Manually add a new profile or change one of non-global profiles, click save btn and verify that client was successfully saved in the table list (formView tab)
3. Go to Realm Settings > Client policies > Profiles >
4. Click "Create new profile" btn and complete all form fields, click "Save", verify that: "Executors" text, "Add Executor" button and Action dropdown are now displaying. From "Action" dropdown select "Delete this client profile", select "Delete" on the delete modal and verify that the client is not listed in the clients table (formView tab)

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Review by edewit — APPROVED** [review]

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: cypress/integration/realm_settings_test.spec.ts [hunk 1]
```diff
@@ -446,28 +446,61 @@ describe("Realm settings tests", () => {
...
-    it("Check deleting the client profile", function () {
+    it("Check deleting the client profile", () => {
       realmSettingsPage.shouldDeleteClientProfileDialog();
     });
+
+    it("Check navigating between Form View and JSON editor", () => {
+      realmSettingsPage.shouldNavigateBetweenFormAndJSONView();
+    });
+
+    // Test works locally but not in CI
```
[the remark this answers is not in the document]
Comment (agagancarczyk) [reply cypress/integration/realm_settings_test.spec.ts:477]: I can try that alright! thanks @jonkoops

File: src/realm-settings/NewClientProfileForm.tsx [hunk 2]
```diff
@@ -46,6 +51,10 @@ export const NewClientProfileForm = () => {
   >([]);
   const [profiles, setProfiles] = useState<ClientProfileRepresentation[]>([]);
   const [showAddExecutorsForm, setShowAddExecutorsForm] = useState(false);
+  const [createdProfile, setCreatedProfile] =
+    useState<ClientProfileRepresentation>({});
```
[the remark this answers is not in the document]
Comment (agagancarczyk) [reply src/realm-settings/NewClientProfileForm.tsx:55]: yeah, sounds good! ty

File: src/realm-settings/ProfilesTab.tsx [hunk 3]
```diff
@@ -112,6 +111,43 @@ export const ProfilesTab = () => {
...
+          .filter((profile: ClientProfile) => !profile.global)
+          .map((profile: ClientProfileRepresentation) =>
+            omit(profile, "global")
+          );
+
+        const changedGlobalProfiles = obj
+          .filter((profile: ClientProfile) => profile.global)
+          .map((profile: ClientProfileRepresentation) =>
+            omit(profile, "global")
+          );
```
[the remark this answers is not in the document]
Comment (agagancarczyk) [reply src/realm-settings/ProfilesTab.tsx:130]: that's spot on! ty

File: src/realm-settings/NewClientProfileForm.tsx [hunk 4]
```diff
@@ -77,14 +86,57 @@ export const NewClientProfileForm = () => {
...
+        (profile) => profile.name !== cdProfile.name
+      );
+
+      try {
+        await adminClient.clientPolicies.createProfiles({
+          profiles: updatedProfiles,
+          globalProfiles,
+        });
+        addAlert(t("deleteClientSuccess"), AlertVariant.success);
+        history.push(`/${realm}/realm-settings/clientPolicies`);
```
[the remark this answers is not in the document]
Comment (agagancarczyk) [reply src/realm-settings/NewClientProfileForm.tsx:112]: It doesn't seem possible. I have tried to add <Link to={toClientPolicies({ realm })}></Link>, but I'm getting this error: ClientPolicies.js:5 Uncaught ReferenceError: Cannot access 'ProfilesTab' before initialization

File: src/realm-settings/NewClientProfileForm.tsx [hunk 5]
```diff
@@ -133,41 +186,44 @@ export const NewClientProfileForm = () => {
...
+                    />
+                  </Text>
+                </FlexItem>
+                <FlexItem align={{ default: "alignRight" }}>
+                  <Button
+                    id="addExecutor"
+                    component={(props) => (
+                      <Link
+                        {...props}
+                        to={`/${realm}/realm-settings/clientPolicies`}
```
[the remark this answers is not in the document]
Comment (agagancarczyk) [reply src/realm-settings/NewClientProfileForm.tsx:213]: This is fixed, ty @jonkoops

File: src/realm-settings/NewClientProfileForm.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (jonkoops) [reply src/realm-settings/NewClientProfileForm.tsx:111]: Ok, then let's leave it for now :+1:

File: src/realm-settings/NewClientProfileForm.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (jonkoops) [reply src/realm-settings/NewClientProfileForm.tsx:212]: No problem, we will look at this some other time

File: src/realm-settings/ProfilesTab.tsx [hunk 6]
```diff
@@ -112,6 +111,43 @@ export const ProfilesTab = () => {
     );
   }
 
+  const normalizeProfile = (
```
Comment (jonkoops) [inline src/realm-settings/ProfilesTab.tsx:114]: You can also use this function in the delete dialog [here](https://github.com/keycloak/keycloak-admin-ui/blob/9f74584078a2472c6f22c94d6b916d4fea0044a0/src/realm-settings/ProfilesTab.tsx#L85).

**Review by jonkoops — APPROVED** [review]
Besides this one comment the PR looks good to me, so I am approving it.

File: src/realm-settings/ProfilesTab.tsx (hunk 6 above)
Comment (agagancarczyk) [reply src/realm-settings/ProfilesTab.tsx]: true, done, thanks @jonkoops

**Pull request merged**